src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logoButton = document.querySelector('.logo');
+
+const promise1 = new Promise(resolve => {
+  logoButton.addEventListener('click', () => {
+    const div1 = document.createElement('div');
+
+    div1.classList.add('message');
+    div1.textContent = 'Promise was resolved!';
+    document.body.appendChild(div1);
```

Comment:
Move it to `then` of the first promise.
Here you should call `resolve`
```suggestion
  resolve();
```